lib/jake/array.ex

```diff
@@ -1,5 +1,49 @@
 defmodule Jake.Array do
-  def gen(_) do
-    StreamData.constant([])
+  def gen(spec) do
+    items = Map.get(spec, "items", %{})
+    uniq = Map.get(spec, "uniqueItems", false)
+    additional_items = Map.get(spec, "additionalItems", %{})
+    max_items = Map.get(spec, "maxItems", 100)
+    min_items = Map.get(spec, "minItems", 0)
+
+    case {items, uniq, additional_items} do
+      {items, true, _} when items == %{} ->
+        StreamData.string(:ascii)
+        |> StreamData.uniq_list_of(max_length: max_items, min_length: min_items)
+
+      {items, false, _} when items == %{} ->
+        StreamData.string(:ascii)
+        |> StreamData.list_of(max_length: max_items, min_length: min_items)
+
+      {items, _uniq, false} when is_list(items) ->
+        items |> Enum.map(&Jake.gen(&1)) |> StreamData.fixed_list()
+
+      {items, _uniq, additional_items} when is_list(items) ->
+        fixed_items_stream = items |> Enum.map(&Jake.gen(&1)) |> StreamData.fixed_list()
+
+        additional_items_stream =
```

Comment:
btw, I prefer it called as `_generator` than `_stream`. A generator can behave like a stream, but not vice-versa.